justfile

```diff
@@ -106,9 +106,17 @@ rust-lld := "" #if use-cargo-zigbuild != "" {
 
 # ICF: link-time identical code folding
 #
-# On windows it works out of the box and on other targets it uses
+# On windows it works out of the box and on linux it uses
 # rust-lld.
-rustc-icf := if for-release != "" { " -C link-arg=-Wl,--icf=safe" } else { "" }
+rustc-icf := if for-release != "" {
+    if target-os != "macos" {
```

Comment:
in that case let's explicit it for linux here, so it matches the comment

Reply:
Thanks, I've fixed this.